GamepadEventDrivenInputAPI/explainer.md

```diff
@@ -0,0 +1,251 @@
+
+### Proposed IDL
+```
+interface GamepadChangeEvent : Event {
+  readonly attribute Gamepad gamepadSnapshot;
+  readonly attribute FrozenArray<long> axesChanged;
+  readonly attribute FrozenArray<long> buttonsChanged;
+  readonly attribute FrozenArray<long> buttonsPressed;
+  readonly attribute FrozenArray<long> buttonsReleased;
+
+  sequence<GamepadChangeEvent> getCoalescedEvents();
+};
+
+```
+### How it works:
+To avoid firing too many events in quick succession for performance issues, the browser may choose to delay firing the gamepadchange event. When this happens, the browser adds the event to an internal queue.
+
+Before running animation callbacks (e.g., `requestAnimationFrame`), the event queue is flushed. This means that all the events that have been delayed will be combined into one single event, representing the union of all changes up to that point.
+
+The final, combined gamepadchange event represents the combined state of the gamepad from all the events that were delayed and coalesced. When this event is dispatched, it contains all the changes that occurred during the delayed period.
+
+## Accessibility, Privacy, and Security Considerations
+To prevent abuse and fingerprinting, a ["gamepad user gesture"](https://www.w3.org/TR/gamepad/#dfn-gamepad-user-gesture) will be required before `RawGamepadInputChange` events start firing (e.g., pressing a button).
+
+Limit Persistent Tracking (fingerprinting): `rawgamepadinputchange` event will not expose any new state that is not already exposed by polling [Fingerprinting in Web](https://www.w3.org/TR/fingerprinting-guidance/).
+
+## Stakeholder Feedback / Opposition
+Firefox: No Signal
+
+Safari: No Signal
+
+Web Developers: Positive
+
+## References & acknowledgements
+
+Thanks to the following contributors and prior work that influenced this proposal:
+
+Firefox’s experimental implementation.
+
+Chromium Prior discussions on improving gamepad input handling.
+
+Many thanks for valuable feedback and advice from:
+- [Steve Becker](https://github.com/SteveBeckerMSFT)
+- [Gabriel Brito](https://github.com/gabrielsanbrito)
+- [Matt Reynolds](https://github.com/nondebug)
+
+Thanks to the following proposals, projects, libraries, frameworks, and languages
+for their work on similar problems that influenced this proposal.
+
+- [Original Proposal](https://docs.google.com/document/d/1rnQ1gU0iwPXbO7OvKS6KO9gyfpSdSQvKhK9_OkzUuKE/edit?pli=1&tab=t.0)
```

Comment:
nit: Let's move this link to link 241 where the prior discussions are described.